packages/dashboard/src/karma/fixture.js

```diff
@@ -87,6 +87,8 @@ const defaultConfig = {
     currentUser: '/wp/v2/users/me',
     fonts: '/web-stories/v1/fonts',
     settings: '/wp/v2/settings',
+    search: '/wp/v2/search',
+    pages: '/wp/v2/pages',
```

Comment:
These features are massively out of date, we haven't had a fonts api in years!

Reply:
I totally forgot that we even had a fonts API!